oneflow/python/ops/array_ops.py

```diff
@@ -2427,3 +2427,123 @@ def ones_job() -> tp.Numpy:
         dtype = flow.float32
 
     return flow.constant(value=1.0, shape=shape, dtype=dtype, name=name + "constant")
+
+
+@oneflow_export("ndindex_to_offset")
+def ndindex_to_offset(
+    index: remote_blob_util.BlobDef,
+    dims: remote_blob_util.BlobDef,
+    name: Optional[str] = None,
+) -> remote_blob_util.BlobDef:
+    """This operator computes the 1-D offset in N-D Tensor.
```

Comment:
这种说法不够直观，可以参考numpy.unravel_index 的说法（它的问题是 `flat indices` 和 `tuple of coordinate arrays` 又是需要先了解的新概念）：
```
Converts a flat index or array of flat indices into a tuple of coordinate arrays.
```

```suggestion
    """This operator converts the n-dimensions index of Tensor to a flatten offset.
```

offset_to_ndindex 同理